src/core/imap.ts

```diff
@@ -39,8 +57,12 @@ export class IMAP {
         });
     }
 
-    setQuery(query: Query) {
-        this.query = query;
+    setEmailMonitoring(query: Query) {
```

Comment:
it's not clear what's the difference between this and mailBoxMonitoring

Reply:
I hope the illustration will explain the difference between them.
![set](https://user-images.githubusercontent.com/58605547/188291510-6821bcf8-9993-4fd6-ac0f-c33b2c44217a.png)

Reply:
I mean I don't understand how these are different functionally. Does one read all the messages and the other only a single mailbox?

Reply:
The first function searches for all new or unread messages that are in the inbox and have been sent from the email that was entered.

The second function searches for all new or unread messages in the mailbox specified by the user.